ckanext/saml2/templates/user/read_base.html

```diff
@@ -0,0 +1,40 @@
+{% ckan_extends %}
+
+{% block user_info %}
+    <div class="info">
+        <dl>
+            {% if c.is_myself or h.check_access('sysadmin') %}
+                <dt>{{ _('GEN: ') }}</dt>
+                <dd>{{ h.saml2_get_user_gen(user.name, true) }}</dd>
+            {% endif %}
+        </dl>
```

Comment:
Do you have any particular reason(design from Dante or client) that forces you to completely rewrite this block, instead of adding GEN slightly above|below, which allow to use `super()`?